test/excel.test.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-env mocha */
+const assert = require('assert');
+const { main } = require('../src/index');
+
+describe('Excel Tests', () => {
+  it('Works for Excel Feeds', async () => {
+    const result = await main({
+      __ow_path: '/https://adobe.sharepoint.com/sites/TheBlog/_layouts/15/guestaccess.aspx',
+      share: 'ESR1N29Z7HpCh1Zfs_0YS_gB4gVSuKyWRut-kNcHVSvkew',
+      email: 'helix%40adobe.com',
+      e: 'vmQSij',
```

Comment:
what is this?

Reply:
Don't know, just part of the share link.

Reply:
usually, the `share link` button in onedrive adds a lot of crap. usually, only the `share` is needed.